tests/NLog.UnitTests/LayoutRenderers/RegistryTests.cs

```diff
@@ -70,6 +103,40 @@ public void RegistryNamedValueTest()
             AssertDebugLastMessage("debug", "FooValue");
         }
 
+#if !NET3_5
+
+        [Fact]
+        public void RegistryNamedValueTest_hive32()
+        {
+            LogManager.Configuration = CreateConfigurationFromString(@"
+            <nlog>
+                <targets><target name='debug' type='Debug' layout='${registry:key=HKCU\\Software\\NLogTest:value=view32:view=Registry32}' /></targets>
+                <rules>
+                    <logger name='*' minlevel='Debug' writeTo='debug' />
+                </rules>
+            </nlog>");
+
+            LogManager.GetLogger("d").Debug("zzz");
+            AssertDebugLastMessage("debug", "reg32");
```

Comment:
I found a difference. `AssertLayoutRendererOutput` only renders the layout renderer, while these test cases really writes to the registry. So keep it like this